internal/api/provider/oidc_test.go

```diff
@@ -81,3 +111,75 @@ func TestParseIDToken(t *testing.T) {
 		require.Equal(t, user.Emails[0].Verified, true)
 	}
 }
+
+func TestAzureIDTOkenClaimsIsEmailVerified(t *testing.T) {
```

Comment:
/nit for searchability. Unless there's an existing name collision
```suggestion
func TestAzureIDTokenClaimsIsEmailVerified(t *testing.T) {
```